reconcile/terraform_resources.py

```diff
@@ -673,17 +689,29 @@ def run(
     use_jump_host=True,
     light=False,
     vault_output_path="",
-    account_name=None,
+    account_name: Optional[tuple[str]] = None,
```

Comment:
any particular reason to use tuple here instead of a generic collection type like `Sequence` or `Iterable`?

Reply:
I think this is safer, because AFAIK we can't change the tipe of  `click` inputs.

Reply:
What do you mean by safer?

The type of the click Param is one thing. The type of the run function can be whatever we want.

Reply:
I just saw that you changed it to collection. Works for me.